vaadin-components-gwt/src/main/java/com/vaadin/components/grid/GridComponent.java

```diff
@@ -288,14 +288,18 @@ public IndexBasedSelectionModel getSelectionModel() {
         return (IndexBasedSelectionModel) grid.getSelectionModel();
     }
 
-    public void refresh() {
+    public void forceRefresh() {
```

Comment:
Agree, I would either leave last API or remove parameter to refresh so as finally we have refresh() and forceRefresh()